src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -241,7 +247,7 @@ public static IFullNodeBuilder UsePoAConsensus(this IFullNodeBuilder fullNodeBui
                             .AddSingleton<INetworkDifficulty, ConsensusQuery>(provider => provider.GetService<ConsensusQuery>())
                             .AddSingleton<IGetUnspentTransaction, ConsensusQuery>(provider => provider.GetService<ConsensusQuery>());
 
-                        new PoAConsensusRulesRegistration().RegisterRules(fullNodeBuilder.Network.Consensus);
+                        new PoAConsensusRulesRegistration().RegisterRules(services);
```

Comment:
Did we need to register rules with an instance of a class?
Couldn't this just be an `IServiceCollection` extension that can be called like
`services.RegisterPoAConsensusRules();` ?

Reply:
No this is a hack to by pass what was there before, I expect a PR to follow that will properly register the rules for Cirrus, the class `PoAConsensusRulesRegistration` should disappear.